js/reducers/navigation.js

```diff
@@ -4,22 +4,26 @@ import { type NavigationState, NavigationActions } from 'react-navigation'
 
 import { type Action } from '../actions/types'
 import { USER_LOGGED_IN_ACTION, USER_LOGGED_OUT_ACTION } from '../actions/login'
+import { LOGIN_SUCCESS } from '../store/actions/constants'
 import AppNavigator from '../navigation/AppNavigator'
-
 const INITIAL_STATE = AppNavigator.router.getInitialState()
 
 const reducer = (
   state: NavigationState = INITIAL_STATE,
   action: Action
 ): NavigationState => {
+  let nextState: ?NavigationState
```

Comment:
`nextState` is actually not needed, I liked more the previous version where the value was returned within the case blocks, why you changed it?

Reply:
what about [this](https://github.com/teamdigitale/italia-app/pull/73)